packages/core/BaseFeatureWidget/BaseFeatureDetail.tsx

```diff
@@ -499,3 +484,18 @@ export const FeatureDetails = (props: {
     </BaseCard>
   )
 }
+
+export default observer((props: BaseInputProps) => {
```

Comment:
Can we make this default export have a name? (e.g. `const BaseFeatureDetails = ...; export default BaseFeatureDetails`). I've noticed without a name components will show up as "anonymous" in React Dev Tools. Also looks like this might apply a few other places in this PR.

Reply:
that should be ok, I would just prefer to keep the observer close to the component, but it can be declared a variable and then exported as a separate statement. i can go and do a look through on this

Reply:
There is a little bit of stylistic inconsistency around this whether a component should by "instantly wrapped" by observer or not. It can save horizontal whitespace to not wrap it instantly, and many components don't instantly wrap it, but I opted to change a bunch of instanes of "export default observer(() => {})" with a named replacement e.g. `const Component = observer(() => {}); export default Component` or `function Component() { ... }; export default observer(Component);`

Reply:
might be resolved now...